hugo/content/contribute.md

```diff
@@ -3,83 +3,279 @@ title: Contribute
+- [Contributing Guidelines](https://github.com/mumble-voip/mumble#contributing)
 
-If you find any issues with them please check if the issue has already been reported, add details if you have more, or add a new tracker item if it has not been reported yet.
+- (Work-in-Progress) [C++ Guidelines & tools](https://github.com/mumble-voip/mumble/discussions/4261) 
+
+### Open Issues and ongoing Development
+
+Take a look at [Issues](https://github.com/mumble-voip/mumble/issues) and [Pullrequests](https://github.com/mumble-voip/mumble/pulls) on Github, especially with the following Labels:
+
+- [Good first issue](https://github.com/mumble-voip/mumble/labels/good%20first%20issue)
+- [Help wanted](https://github.com/mumble-voip/mumble/labels/help%20wanted)
+- [Feature Request](https://github.com/mumble-voip/mumble/labels/feature-request)
+
+<!-- mention priority labels, e.g. [P2- Critical](https://github.com/mumble-voip/mumble/labels/priority%2FP2%20-%20Important) -->
+
+Under [Milestones](https://github.com/mumble-voip/mumble/milestones) we list the Issues that are planned to be worked on for the next releases.
+
+
+*Tips:*
+
+- To avoid unnecessary work and duplication, comment on Issues and inform others what you plan to work on.
+
+
+<!-- old:
+
+We are always looking for helping hands willing to contribute to mumble. You'll need a good understanding of C++ and previous knowledge in Qt is definitely helpful. The best way to get into mumble development is to link up with us on {{< wiki IRC />}} ([#mumble-dev on freenode](irc://chat.freenode.net/mumble-dev)), there is always small stuff to do to get you started. Also check the {{< wiki Development "Development wiki page" />}} on more info about the source code and building it.
 
-# Translation
+-->
+
+### Additional projects
+
+#### 3rd party Interfaces
+
+<!-- mention gRPC? -->
+
+Web interfaces, web-viewers, administration programs, CMS website modules – all neat things we would love to see! These could be written in any language supported by our Ice based RPC Interface. You could start your own project or link up with an {{< wiki "3rd Party Applications" "already existing one" />}}. To get an idea about how interaction with Murmur might look like you can take a look at the example scripts in our git repositories script folder.
+
+#### Ice authenticators
+
+See {{< wiki Authenticators />}}. We’re happy for every authenticator and authenticator improvement you and the community can bring us!
+
+#### Positional Audio
+
+Most of our positional audio plugins regularly break on game updates. To read on how you can help us update the plugins see the {{< wiki Pluginguide />}} page. You can even fix the update-issue forever for open source games, or if you want to add positional audio to your game, see {{< wiki Link />}}.
+
+<!-- mention third-party-projects? -->
```

Comment:
Do you mean things like e.g. grumble?

Reply:
Well, strictly seen grumble is an internal project or not?
But still it could be mentioned as well.

There are a lot of projects: pymumble, Webrtc bridge, Bots etc.

Reply:
Yeah strictly speaking that's true. As nobody from the team is working on it though I consider it to be pretty much 3rdParty by now ^^

Imo we can leave 3rdParty apps out here as this page is supposed to be about contributing to the Mumble project. From my point of view that refers to the "official" components only.
I think we should have a separate page for 3rdParty applications with links to their docs. If someone wants to contribute there, they can find the necessary information at that project's website :point_up:

Reply:
I would have only mentioned them, like "hey if you are interested in more projects, here are some projects related to Mumble".

Reply:
In that case I think we can simply link to the 3rdParty apps page from the contribute page for this purpose.

Reply:
Thats the idea.

Reply:
I just noticed that 3rd party apps are mentioned in `3rd party Interfaces`...
I think it's not the same, also because it mentions a focus on Ice...

I will add small part mentioning them with a link.